components/profile/payments-list.tsx

```diff
@@ -1,16 +1,74 @@
-import PaymentItem from "components/profile/payment-item";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+
+import Button from "components/button";
+
+import { formatNumberToCurrency } from "helpers/formatNumber";
 
 import { Payment } from "interfaces/payments";
 
+import useNetworkTheme from "x-hooks/use-network";
+
+import { FlexColumn, FlexRow } from "./wallet-balance";
+
 interface PaymentsListProps {
   payments: Payment[];
 }
 
 // TODO: Add InfiniteScroll and pagination
 export default function PaymentsList({ payments } : PaymentsListProps) {
+  const { push } = useRouter();
+  const { t } = useTranslation(["common", "profile", "bounty"]);
+  const { getURLWithNetwork } = useNetworkTheme();
+
+  function PaymentItem({ ammount, issue, transactionHash, id} : Payment) {
```

Comment:
filename is called "payments list", it should only contain ... payments-list. there was a file named "payment-item" because we could want to use that same component elsewhere :)

Reply:
i had a problem to use the translations in the payment-item component this was the solution that solved this problem. But I can investigate further and see other solutions like passing the translation